manifests/image-references

```diff
@@ -42,3 +42,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-powervs-cloud-controller-manager
+  - name: nutanix-cloud-controller-manager
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-nutanix-cloud-controller-manager
```

Comment:
This must not be merged until the image is in payload, you may want to split this into a separate PR to get the other parts of this PR merged earlier.

Reply:
Now the nutanix-ccm image is in payload.